### PR TITLE
Fix bug with unaliased named columns

### PR DESCRIPTION
**Pull request by rschlussel-zz**
Make unaliased named columns have aliased set to false in DESCRIBE
OUTPUT.

Testing: added unit test

@petroav

### CODE REVIEW

**Comment by petroav** [comment]
LGTM, thanks for taking this on. Feel free to squash this with my original commit https://github.com/Teradata/presto/commit/24109b7b4ece3df22c721975ef3cf262c935b1e7

File: presto-main/src/main/java/com/facebook/presto/sql/analyzer/StatementAnalyzer.java [hunk 1]
```diff
@@ -1714,13 +1715,13 @@ else if (expression instanceof DereferenceExpression) {
...
-                if (!alias.isPresent()) {
+                if (!fieldName.isPresent()) {
                     if (name != null) {
-                        alias = Optional.of(getLast(name.getOriginalParts()));
+                        fieldName = Optional.of(getLast(name.getOriginalParts()));
                     }
                 }
 
-                outputFields.add(Field.newUnqualified(alias, analysis.getType(expression), qualifiedOriginTable, alias.isPresent())); // TODO don't use analysis as a side-channel. Use outputExpressions to look up the type
+                outputFields.add(Field.newUnqualified(fieldName, analysis.getType(expression), qualifiedOriginTable, isAliased)); // TODO don't use analysis as a side-channel. Use outputExpressions to look up the type
```
Comment (rschlussel-zz) [inline presto-main/src/main/java/com/facebook/presto/sql/analyzer/StatementAnalyzer.java:1724]: you had a good point about the method being confusing.  I'm going to change this to call column.getAlias().isPresent() right here so it's clear where it's coming from.

Comment (petroav) [inline presto-main/src/main/java/com/facebook/presto/sql/analyzer/StatementAnalyzer.java:1724]: That's an even better solution!

**Comment by rschlussel-zz** [comment]
merged. (squashed into describe output commit)

**Pull request closed without merging**